spec/sidekiq/evss/disability_compensation_form/submit_form526_all_claim_spec.rb

```diff
@@ -58,6 +57,8 @@
       cassettes.each { |cassette| VCR.insert_cassette(cassette) }
       Flipper.disable(ApiProviderFactory::FEATURE_TOGGLE_RATED_DISABILITIES_BACKGROUND)
       Flipper.disable(ApiProviderFactory::FEATURE_TOGGLE_RATED_DISABILITIES_FOREGROUND)
+      allow(Flipper).to receive(:enabled?).with(:disability_526_migrate_contention_classification,
```

Comment:
Can you please update all the Flipper.disable to:
 allow(Flipper).to receive(:enabled?).with(:flipper_name).and_return(false)
 
 If you are receiving testing errors from this, please feel free to reach out to me.